pkg/resource/events-logger-config/grafana-agent/events-logger.grafanaagent.template

```diff
@@ -1,17 +1,35 @@
-# This file was generated by logging-operator.
```

Comment:
I think this is wrong, you should use the versions in main for the 2 templates :)

Reply:
Thé content of this file and thé other is reversed in your PR.

Reply:
Or not, I think it mixed them up :(

Reply:
You'll have to regenerate thé files using `go test ./... -update`

Reply:
> Thé content of this file and thé other is reversed in your PR 

That's because in your smaller PR you inverted the template files' content and I missed that when reviewing 

> You'll have to regenerate thé files using go test ./... -update

Ok :ok_hand:

Reply:
Yes I mixed them up sorry :(

Reply:
@QuentinBisson can you update the tests on your side ? I get the following errors : 
```
[...]
# github.com/klauspost/compress/internal/cpuinfo
compile: version "go1.22.0" does not match go tool version "go1.23.3"
FAIL    github.com/giantswarm/logging-operator [build failed]
FAIL    github.com/giantswarm/logging-operator/internal/controller [build failed]
[...]
```

Reply:
Yeah but : 
```
$ go version
go version go1.23.3 linux/amd64
```

Reply:
I'll gen then in anothe PR then otherwise I cannot review this one. But we need to figure out what's failing on your machine